IOhandler.cs

```diff
@@ -232,7 +238,6 @@ public void updateVehicleWithAnyExtraFields(GarageLogic.Vehicle i_Vehicle)
 
             foreach (FieldInfo memberField in vehicleMembers)
             {
```

Comment:
Do not remove 
//string nameOfMemberField = getNameOfMemberField(memberField);

We still might need it